pc/CODEC2 GUI/CODEC2 GUI/MainForm.cs

```diff
@@ -86,8 +86,11 @@ private void Form1_Load(object sender, EventArgs e)
 
             API.RadioAdded += new API.RadioAddedEventHandler(API_RadioAdded);
             API.RadioRemoved += new API.RadioRemovedEventHandler(API_RadioRemoved);
+            API.IsGUI = true;
```

Comment:
I just added this and I had SmartSDR running first then this GUI ran ok.
I can take it out. Let me know.


Regards,
Mark

On Aug 31, 2015, at 7:27 PM, Stephen Hicks <notifications@github.com> wrote:

In pc/CODEC2 GUI/CODEC2 GUI/MainForm.cs:

> @@ -86,8 +86,11 @@ private void Form1_Load(object sender, EventArgs e)
>  
>              API.RadioAdded += new API.RadioAddedEventHandler(API_RadioAdded);
>              API.RadioRemoved += new API.RadioRemovedEventHandler(API_RadioRemoved);
> +            API.IsGUI = true;
I think this will prevent SmartSDR from running since only a single GUI client is allowed. The "GUI Client" is the one that shows panadapters/waterfalls. Does this actually run like this?

—
Reply to this email directly or view it on GitHub.